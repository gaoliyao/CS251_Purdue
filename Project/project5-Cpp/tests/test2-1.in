2
50 300
AAA AAW 332.48
AAB AAY 832.45
AAC AAO 727.93
AAD ABX 595.61
AAE AAA 270.49
AAF ABE 961.06
AAG AAE 814.24
AAH AAT 504.88
AAI ABR 682.88
AAJ ABW 547.81
AAK AAM 674.00
AAL AAN 612.09
AAM AAX 167.26
AAN ABP 548.14
AAO ABM 560.38
AAP ABE 602.13
AAQ ABK 824.56
AAR AAK 112.51
AAS ABX 811.12
AAT AAO 778.02
AAU AAN 254.94
AAV AAI 891.19
AAW AAV 1.30
AAX AAH 947.14
AAY AAS 757.57
AAZ ABV 692.30
ABA AAH 555.10
ABB ABP 433.81
ABC ABM 401.34
ABD ABV 100.79
ABE AAW 500.27
ABF ABQ 854.59
ABG ABK 464.54
ABH AAM 40.85
ABI AAQ 2.26
ABJ ABN 684.31
ABK AAX 885.00
ABL AAA 816.96
ABM ABE 971.57
ABN AAK 570.48
ABO AAQ 422.58
ABP ABE 324.24
ABQ AAU 894.21
ABR AAH 27.09
ABS AAH 67.04
ABT ABM 420.80
ABU ABI 994.32
ABV ABP 219.20
ABW ABM 733.05
ABX AAH 61.57
ABF ABV 745.88
AAG ABV 321.39
AAC ABB 456.48
AAP ABB 349.15
AAZ AAH 836.91
ABV ABE 758.48
AAI ABL 989.04
ABW AAT 719.88
ABG AAM 102.60
AAV AAR 689.70
ABC AAB 95.67
AAQ AAH 550.16
AAF AAM 890.95
AAE ABB 965.74
AAV AAE 466.83
AAM AAB 862.13
AAC AAQ 708.08
AAY AAF 302.91
ABK AAR 683.64
AAE ABL 962.42
AAF AAG 150.60
AAO AAY 496.66
ABA AAT 462.72
AAF ABR 941.45
AAJ ABA 93.57
AAT AAX 104.46
ABJ ABH 586.70
ABQ ABA 273.58
AAK ABO 571.12
ABX ABG 705.19
AAL AAM 605.02
ABJ AAS 336.97
ABE ABD 535.76
ABN ABQ 596.67
AAI AAR 701.13
ABT AAB 451.35
ABM ABB 724.94
AAA AAU 459.58
ABX AAC 164.78
AAH ABV 160.14
ABQ AAI 738.28
ABS ABT 255.98
ABR ABB 883.78
AAT AAQ 74.22
ABL ABA 98.42
AAC AAW 664.45
ABM AAK 238.74
ABI AAM 504.94
ABQ AAN 425.47
AAU AAX 314.44
ABK AAP 832.05
AAR AAJ 604.52
AAP AAC 409.80
ABT AAY 504.96
AAP ABT 601.07
AAF AAH 839.81
ABN AAT 344.76
ABG ABI 770.23
AAE AAH 248.19
ABR AAW 243.76
AAK ABH 848.29
AAZ ABM 258.09
AAZ ABE 204.05
ABQ ABB 805.12
AAA ABI 808.45
ABN AAF 153.21
AAX ABQ 86.96
ABE ABX 498.68
AAZ AAW 742.44
ABJ AAH 754.25
AAM ABT 12.34
ABQ ABE 163.84
AAW ABJ 578.77
AAA ABX 823.77
ABK ABT 346.18
ABS AAS 52.94
AAY ABL 625.83
ABX AAL 368.27
ABK AAT 122.53
ABW AAO 298.39
ABK AAY 237.31
AAS AAK 487.56
AAI AAF 632.16
ABU AAC 141.87
ABQ AAV 194.81
ABC ABK 374.07
AAS AAV 494.58
ABD ABW 668.44
ABW AAD 887.94
ABG ABP 658.76
ABI AAH 763.91
ABS AAO 396.07
ABP AAR 701.46
ABH ABM 59.79
AAO ABA 433.86
ABG ABV 91.96
AAL ABV 760.40
AAM AAD 811.87
ABB ABW 557.71
AAS ABA 740.24
AAT AAL 305.81
AAK AAF 659.98
AAU ABW 199.15
AAS ABG 290.55
ABF ABI 279.30
ABC ABF 977.43
ABB AAX 617.05
ABE AAM 407.34
ABC ABI 923.52
ABQ ABK 497.25
AAY AAH 576.77
AAI AAM 296.75
AAG ABT 614.11
ABQ ABH 984.02
AAS ABP 579.83
ABG AAZ 633.13
AAQ ABE 802.70
ABO AAV 858.42
AAQ AAN 775.01
ABQ AAW 351.78
AAC ABK 812.05
AAK ABF 426.16
AAA AAG 449.61
ABD ABA 549.45
AAL AAC 182.59
ABE ABI 148.81
AAJ ABM 710.72
ABO ABV 521.33
ABI AAT 36.63
ABK AAH 848.68
ABU ABM 274.85
ABW ABU 724.46
ABF AAG 530.77
AAI AAE 838.90
AAM AAP 515.22
ABH ABK 429.20
AAP AAS 362.60
AAO AAE 397.44
ABJ AAR 990.32
ABC ABE 974.23
ABA ABC 862.21
AAK ABL 556.50
AAU ABP 395.40
ABG AAG 74.15
ABC AAE 240.48
AAQ AAZ 945.14
ABI ABL 301.80
ABM ABG 705.16
AAI AAO 429.58
ABO ABH 842.56
ABO AAF 560.97
ABG ABL 849.05
ABV AAR 692.10
ABW ABH 127.87
AAO AAU 926.24
ABS AAK 285.58
AAZ AAT 592.14
AAU AAK 205.33
ABT ABD 844.25
ABF ABK 39.67
ABK ABJ 794.02
AAY ABP 622.51
ABR AAD 691.49
ABD ABQ 869.13
AAB AAS 348.58
AAL ABB 533.18
ABG AAE 310.89
ABU ABA 961.49
ABG AAV 796.29
AAJ AAC 637.17
ABD ABN 39.02
ABF AAP 453.45
ABG AAI 723.50
ABI ABN 284.94
AAN AAH 403.54
ABQ ABC 828.36
ABU AAR 8.84
AAW AAT 558.18
ABR AAA 951.78
AAH ABL 181.71
ABO AAC 635.17
AAW AAM 522.19
AAI AAC 807.14
AAI ABS 579.77
AAP ABW 289.19
ABT ABI 570.65
AAN AAV 33.39
AAR ABE 532.51
ABA AAR 714.22
ABI AAL 152.99
ABK ABB 202.07
ABE AAE 664.25
ABF AAZ 72.70
AAP ABQ 901.85
AAP AAD 899.45
ABU AAG 521.72
AAP ABD 246.88
ABD AAV 788.33
ABF AAH 794.10
ABM ABN 543.16
AAP AAW 581.41
ABK ABO 517.35
AAX ABA 323.16
ABR AAE 470.32
ABW AAC 200.30
AAF ABA 287.95
ABK AAO 892.32
AAI ABU 276.53
ABS ABB 504.62
AAG AAR 572.54
AAX ABJ 879.76
AAQ AAX 817.66
ABT AAU 52.41
AAB AAE 731.90
AAD ABI 937.47
ABF ABG 219.83
ABU AAO 255.41
ABL AAX 359.82
ABS ABQ 932.36
AAR ABB 975.65
ABJ AAA 793.31
ABN ABX 743.28
ABE AAO 196.82
ABJ AAB 297.81
AAR ABH 517.64
AAP ABV 936.58
AAL AAR 342.95
ABP ABA 742.38
AAI AAG 718.03
ABL ABJ 495.07
AAA ABV 57.62
ABI ABJ 714.65
AAQ AAE 730.98
ABG AAT 325.97
AAR ABO 507.66
AAT ABF 801.44
AAH ABQ 1.52
AAN ABM 247.61
AAC AAX 906.20
AAT AAV 29.72
AAJ AAE 705.00
AAZ AAI 599.50
AAJ ABD 925.47
AAO ABQ 918.27
ABH AAZ 883.23
ABR AAR 884.76
ABD AAC 231.82
ABW ABL 745.41
AAC ABL 40.68
AAM ABL 798.79
ABP ABK
ABH AAC
AAD ABB
ABF ABM
AAD AAF
ABH AAU
ABH ABP
ABA AAW
AAD AAH
ABF ABH
END
