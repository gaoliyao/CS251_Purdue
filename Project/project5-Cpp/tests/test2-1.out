ABP ABV ABG ABF ABK 570.66
ABH ABW AAC 328.17
AAD ABX AAH ABQ AAX AAT ABK ABB 1174.72
ABF AAZ ABM 330.79
AAD ABX AAH ABV ABD ABN AAF 1110.34
ABH AAM ABT AAU 105.60
ABH AAM ABG ABV ABP 454.61
ABA AAX AAT AAV AAW 458.64
AAD ABX AAH 657.18
ABF ABG AAM ABH 363.28
